GVFS/GVFS.Platform.Mac/MacFileSystemVirtualizer.cs

```diff
@@ -355,22 +355,19 @@ private bool TryGetSymLinkTarget(string sha, out string symLinkTarget)
                 metadata.Add(nameof(sha), sha);
                 metadata.Add(nameof(placeholderVersion), placeholderVersion);
                 metadata.Add(nameof(commandId), commandId);
-                ITracer activity = this.Context.Tracer.StartActivity("GetFileStream", EventLevel.Verbose, Keywords.Telemetry, metadata);

```

Comment:
If this is a hot path - should we only create the event metadata if we are going to use it (i.e. inside the following if block)?

Reply:
Good idea. I can move it to a private method to avoid code duplication, too. Thanks!

Reply:
We can do that on the Mac, but not Windows (at least not easily), as there is an async handler that takes the existing metadata and adds to it if necessary.